app/src/pages/admin/projectsPage/projectsPanelView/projectsPanelView.jsx

```diff
@@ -50,19 +52,45 @@ export class ProjectsPanelView extends Component {
     onDelete: () => {},
   };
 
-  getPanelList = (projects) =>
-    projects.map((project) => (
-      <div className={cx('panel')} key={project.id}>
-        <ProjectPanel
-          project={project}
-          onMembersClick={this.props.onMembers}
-          onSettingsClick={this.props.onSettings}
-          onAssign={this.props.onAssign}
-          onUnassign={this.props.onUnassign}
-          onDelete={this.props.onDelete}
-        />
-      </div>
-    ));
+  constructor(props) {
+    super(props);
+    this.state = {
+      organizations: [],
+    };
+
+    fetch(URLS.organizationsList(), {
```

Comment:
Please add a corresponding `TODO:` comment if it is a temporary solution.

Reply:
Done.